dvc/command/version.py

```diff
@@ -0,0 +1,41 @@
+from __future__ import unicode_literals
+
+import platform
+import argparse
+import logging
+
+from dvc.command.base import CmdBase, append_doc_link
+from dvc.version import _BASE_VERSION
+
+
+logger = logging.getLogger(__name__)
+
+
+class CmdVersion(CmdBase):
+    def run(self):
+        info = (
+            "DVC version: {dvc_version}\n"
+            "Python version: {python_version}\n"
+            "Platform: {platform_type}"
+        ).format(
+            dvc_version=_BASE_VERSION,
```

Comment:
Let's use `from dvc import __version__` instead. _BASE_VERSION is not meant for direct use and is not available in installed packages, because we auto-generate version.py.

Reply:
@efiop I used `__version__` initially but that also gives information about git repository. Will that be fine?

Reply:
Yes, that is what it supposed to do, so that we distinguish official releases from something installed by hand.